torch/_inductor/fx_passes/post_grad.py

```diff
@@ -627,44 +630,69 @@ def reinplace_scatters(graph):
     for node in reversed(graph.nodes):
         storage_to_nodes[get_node_storage(node)].append(node)
         if node.target == aten.copy_.default:
-            copy_args_to_copy_nodes[(node.args[0], node.args[1])] = node
+            src = node.args[0]
+            dst = node.args[1]
+            # If the target is a getitem and it indexes a possible clone,
+            # then skip over it
+            if (
+                dst.target == operator.getitem
+                and dst.args[0].kwargs["kwargs"][dst.args[1]] == node.args[0]
+            ):
+                dst = dst.args[0]
```

Comment:
I'm a little iffy about this since `dst` becomes `triton_kernel_wrapper_functional_proxy` here but with above check that should be safe

Reply:
seems a bit confusing to me, since `getitem` is the actual node that we're copying into the input, not `triton_kernel_wrapper_functional_proxy`. But I guess it's not wrong 🤔